crates/rattler_repodata_gateway/src/utils/simple_channel_server.rs

```diff
@@ -45,7 +45,8 @@ impl SimpleChannelServer {
 
         // Spawn the server. Let go of the JoinHandle, we can use the graceful shutdown trigger to
         // stop the server.
-        let _ = tokio::spawn(server);
+        let future = tokio::spawn(server);
```

Comment:
I think this should just be _future instead of dropping.

Reply:
clippy was suggesting this.

Reply:
Yeah its a bug. Assignment to _ immediately drops the value. Which might be counterintuitive. But we do want to keep the value around (I think).

But whatever, the tests still succeed so nvm!